dolphinscheduler-storage-plugin/dolphinscheduler-storage-api/src/main/java/org/apache/dolphinscheduler/plugin/storage/api/StorageOperator.java

```diff
@@ -24,9 +24,6 @@
 
 public interface StorageOperator {
 
-    String FILE_FOLDER_NAME = "resources";
-    String UDF_FOLDER_NAME = "udfs";
```

Comment:
Please Keep these two fields, we should avoid add all constants in Constants, these fields in StorageOperator is also static. You can get these by `StorageOperator.FILE_FOLDER_NAME`

Reply:
The `udfs` have been remove from `ResourceType`.